### ISSUE TITLE
Ssh needed for CSI Monitor GUI on 2 devices?

### ISSUE THREAD

**Issue by teaganharatua** [brunosoaresds/csi-monitor#9]
Hello, is the ssh connection still needed if only two devices were used (1 AP and one Client (both Ubuntu 14.04))? I've been running the csi-gui-tool on the AP device, but I'm unable to start the monitoring using the gui, as nothing shows up.

Thanks again!

**Comment by teaganharatua** [comment]
Attached is the error log in the MATLAB 

![erro_log](https://user-images.githubusercontent.com/35765149/70955121-4a350180-20a2-11ea-849a-9a500c0295f4.png)

**Comment by brunosoaresds** [comment]
Yes, its needed @teaganharatua . But you can use ssh to localhost. That ssh layer is a part of the code, but you can change it on the python layer of realtime_view.

About your print problem, is a rename command error. im fixing it right now on the master branch. Pull the new code and initialize the view by the realtime_view command.

**Comment by teaganharatua** [comment]
Thanks for the reply! another question, if I may @brunosoaresds, After managing to connect into the realtime_view, and changing the MAC Adress to the reciever's MAC Address on "self.socket.send" in the DMClient.py file. I ran the realtime_view GUI

The resulting graph shows a flat line, and the exported log file is showing empty arrays, any idea on the cause of the issue?

Attached are the screenshots of the issue

![image](https://user-images.githubusercontent.com/35765149/71771450-92db2e80-2f6d-11ea-9d13-754d2ac33b32.png)
![image](https://user-images.githubusercontent.com/35765149/71771454-a2f30e00-2f6d-11ea-92e7-d51e73752052.png)
![image](https://user-images.githubusercontent.com/35765149/71771468-d46bd980-2f6d-11ea-8fe4-98d4b7bcf335.png)
